src/psyclone/tests/generator_test.py

```diff
@@ -102,6 +102,7 @@ def test_similar_kernel_name():
 
```

Comment:
Please add your name and copyright info to the top of the file. Again the license information needs changing but that can be done separately.